### PR TITLE
FIX: setup types

### PR DESCRIPTION
**Pull request by hui-zhou-a**
PyEDB is failing on 24R1

### CODE REVIEW

File: src/pyedb/dotnet/edb.py [hunk 1]
```diff
@@ -3609,17 +3609,18 @@ def setups(self):
         Dict[str, :class:`legacy.edb_core.edb_data.siwave_simulation_setup_data.SiwaveSYZSimulationSetup`]
 
         """
+        setup_types = {}
```
Comment (SMoraisAnsys) [inline src/pyedb/dotnet/edb.py:3612]:
```suggestion
```

**Review by SMoraisAnsys — COMMENTED** [review]
I'm not sure to understand what was failing in 2024.1.
Is it because some `SimulationSetupType.whateverthename` do not exist in that version ?
I left a minor comment due to an unused variable.

**Comment by hui-zhou-a** [comment]
> I'm not sure to understand what was failing in 2024.1. Is it because some `SimulationSetupType.whateverthename` do not exist in that version ? I left a minor comment due to an unused variable.

That's right. below property doesn't exist in 2024.1.

`self.edb_api.utility.utility.SimulationSetupType.kRaptorX`

**Comment by SMoraisAnsys** [comment]
> > I'm not sure to understand what was failing in 2024.1. Is it because some `SimulationSetupType.whateverthename` do not exist in that version ? I left a minor comment due to an unused variable.
> 
> That's right. below property doesn't exist in 2024.1.
> 
> `self.edb_api.utility.utility.SimulationSetupType.kRaptorX`

If there is no changes associated to the "name" of the SimulationSetupType in coming AEDT releases then this changes will be great. Otherwise, we might want to keep the previous approach and add extra if condition based on AEDT versions. For example, we could have an if statement like `<=2024.1` before the test on kRaptorX.

**Review by SMoraisAnsys — APPROVED** [review]

**Pull request merged**